riff-raff/test/ci/ContinuousDeploymentTest.scala

```diff
@@ -43,9 +43,9 @@ class ContinuousDeploymentTest extends AnyFlatSpec with Matchers {
   val contDeployConfigs = Seq(tdProdEnabled, tdCodeDisabled, td2ProdDisabled, td2QaEnabled)
   val contDeployBranchConfigs = Seq(tdProdEnabled, tdCodeDisabled, td2ProdDisabled, td2QaBranchEnabled, td2ProdBranchEnabled)
 
-  val tdB71 = S3Build(45397, "tools::deploy", "45397", "branch", "71", new DateTime(2013,1,25,14,42,47), "", "")
-  val td2B392 = S3Build(45400, "tools::deploy2", "45400", "branch", "392", new DateTime(2013,1,25,15,34,47), "", "")
-  val otherBranch = S3Build(45401, "tools::deploy2", "45401", "other", "393", new DateTime(2013,1,25,15,34,47), "", "")
+  val tdB71 = S3Build(45397, "tools::deploy", "45397", "branch", "71", new DateTime(2013,1,25,14,42,47), "", "", None)
```

Comment:
Not blocking, but perhaps:
- Using named parameters though more verbose is a good standard practise (to avoid the kind of issue like the switched params above).
- Filling the extra fields out here with fake data for build tool & repo would make these tests more useful.